lib/libbakery.sh

```diff
@@ -11,7 +11,7 @@ libroot="$(dirname "${BASH_SOURCE[0]}")"
 scriptroot="$(cd "$(dirname "${BASH_SOURCE[0]}")/../"; pwd)"
 
 bakery="flatcar/sysext-bakery"
```

Comment:
We need to take care of this one as well, it is used to query version information of existing images from github to determine which ones to build.